central/metadata/service/service_impl.go

```diff
@@ -184,6 +185,7 @@ func (s *serviceImpl) GetDatabaseStatus(ctx context.Context, _ *v1.Empty) (*v1.D
 		dbStatus.DatabaseType = dbType
 	}
 
+	dbStatus.DatabaseIsExternal = pgconfig.IsExternalDatabase()
```

Comment:
Should go after the DatabaseType and only shown to logged in users.

Reply:
Done.